### PR TITLE
[chore] Add automatic PR labeler workflow

### PR DESCRIPTION
**Pull request by hspinks**
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [x] ~REQUIRED: Do you have an Issue **assigned to you** within the v1 milestone for this PR?  Put the Issue number here:~
- [x] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [x] ~Tested your changes, especially after any code review iterations.~
- [x] ~Included any relevant screenshots of UI updates.~
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

## What does this PR accomplish?

Adds a GitHub action to automatically label PRs opened with relevant labels. Currently supports `client`, `docs`, `ops`, `server`, `string-change`, and `tests` labels (see `.github/labeler.yml` in this PR for the configuration).

## Did you add any dependencies?

* [`actions/labeler`](https://github.com/actions/labeler) GitHub Action added - officially-supported action from the GitHub team

## How did you test the change?

N/A

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+client:
+  - client/flutter/**
```
Comment (advayDev1) [inline .github/labeler.yml:2]: > 
```suggestion
  - client/**
```

**Comment by SamMousa** [comment]
Can we make the flow optional? Ie if it fails due to misconfiguration it shouldn't block merging prs

File: .github/labeler.yml (hunk 1 above)
Comment (hspinks) [reply .github/labeler.yml:2]: @advayDev1 I've been using the `client` tag to refer to the Flutter client to make it easier to direct people to Flutter tasks/PRs to review. Happy to rename `client` to `flutter` if you think that's better, or we can include other clients in the `client` tag

Comment (SamMousa) [reply .github/labeler.yml:2]: Current operating assumption is just 1 client right? So imo just use client.

Comment (hspinks) [reply .github/labeler.yml:2]: the question is if we want changes in the [`client/rpc`](https://github.com/WorldHealthOrganization/app/tree/master/client/rpc) directory also labeled with the `client` tag

Comment (hspinks) [reply .github/labeler.yml:2]: I will accept this change as proposed for now. We don't have a high volume of PRs editing the `client/rpc` directory anyway, so probably not worth discussing further at this point... if that does become an issue, we can always edit this later

**Comment by hspinks** [comment]
Welp... [labeler doesn't work for forks](https://github.com/actions/labeler/commit/2b75d22987baa8b9fb023106fe3a4d90648b7104)... see https://github.com/actions/labeler/issues/12 😞

**Comment by hspinks** [comment]
Closing since GitHub's `labeler` action does not work with PRs opened from forks. Workaround would be to write a different script for the action and use cron in the repo, but that would be a separate issue / PR

**Pull request closed without merging**